standard/classes.md

```diff
@@ -3056,6 +3056,7 @@ When the first parameter of a method includes the `this` modifier, that method i
 
 - It may only be an input parameter if it has a value type
 - It may only be a reference parameter if it has a value type or has a generic type constrained to struct
+- It shall not be an output parameter.
```

Comment:
While we're here, is it worth being consistent within this bullet list as to whether we end with a period or not?